content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Validate Certificate revocation with OCSP
+- Support experimental features
+- Introduce new CLI command `notation policy` for managing trust policy configuration
```

Comment:
How about **Introduce new CLI command `notation policy` to simplify trust policy configuration**